controllers/topics.controllers.js

```diff
@@ -0,0 +1,10 @@
+const fetchTopics = require('../models/topics.models.js');
+
+function getTopics(req,res,next) {
+    fetchTopics()
+    .then((topics) => {
+        res.status(200).send(topics)
```

Comment:
Here is where you should be sending your data in an object with a key of `topics`